doc/Project.toml

```diff
@@ -0,0 +1,5 @@
+[deps]
+Clustering = "aaaa29a8-35af-508c-8bc3-b662a17a0fe5"
```

Comment:
I don't think packages are supposed to be listed in their own doc projects unless a corresponding doc manifest is checked in, since version resolution will install the released version.